### DISCUSSION TITLE
Copilot is not available in your location

### DISCUSSION THREAD

**Discussion by Stepashka20** [community/community#123651] [Copilot Conversations] [answered]
A couple of days ago I started getting an error:
`[error] [auth] Extension activation failed: "At this time, Copilot is not available in your location..."`

I am always using proxy in IDE and everything always worked, but now I'm getting an error. Proxies are working.How to fix it?

**Comment by InWamos** [comment]
Is your country of residence under any sanctions?
Maybe proxies are connected to the country with sanctions

**Reply by Stepashka20** [reply]
Proxies from a country without sanctions, they worked for a very long time

**Reply by InWamos** [reply]
yeah, I just read that GitHub doesn't restrict because of sanctions. Does it throws any other mistakes apart from the one, you stated?

**Reply by davecheney** [reply]
@Stepashka20 @InWamos regrettably Copilot was not in scope when GitHub obtained an exception from the US DFAT. Copilot is restricted in certain countries or regions as required by US law.

**Reply by Stepashka20** [reply]
@davecheney 
I know that the copilot is restricted in Russia, but I have a proxy from another country that is not under sanctions. VSCode with VPN works on this server, but proxies from this server do not work. I think that copilot specifically ignores the proxy, because 1 week ago all was good

**Comment by InWamos** [comment]
> proxy in IDE

People from Russia and Belarus say that they managed to solve the problem by using VPN Instead of proxies

**Comment by andamound** [comment]
The proxy works correctly on v1.188.0, since v1.189.0 the proxy does not work. You can temporarily roll back the extension version

**Reply by Stepashka20** [reply]
Wow thanks!!! 😍

**Reply by devm33** [reply, marked as answer]
Please update the Copilot extension to version 1.192.0 to resolve this issue. Sorry for the trouble here.
